OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -29,8 +29,16 @@ public enum CellConditions
 		All = TransientActors | BlockedByMovers
 	}
 
+	public interface IMobileInfo
```

Comment:
Style nit: It would be a bit cleaner to make this inherit from `IMoveInfo`, and then remove that from `MobileInfo` below.